src/pages/Jiyoung/pages/Login/Login.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Button from '../../components/Button/Button';
+import '../../scss/style.scss';
+
+const Login = () => {
+  const navigate = useNavigate();
+  const [userId, setUserId] = useState('');
+  const [userPw, setUserPw] = useState('');
+  const emailPattern =
+    /^([0-9a-zA-Z_.-]+)@([0-9a-zA-Z_-]+)(\.[0-9a-zA-Z_-]+){1,2}$/;
+
+  const joinBtn = () => {
+    navigate('/jiyoung-join');
+  };
+
+  const saveUserId = (event) => {
+    setUserId(event.target.value);
+  };
```

Comment:
아래와같이 구조분해할당을 통해 코드를 간소화할 수 있습니다.
```suggestion
  const saveUserId = (event) => {
  const {value} = event.target;
    setUserId(value);
  };
```